File: programme/area-of-square/area-of-square.cs
// Computes the area of a square
using System;

public class SquareArea
{
    static void Main(string[] args)
    {
        double SquareArea(string input)
        {
            return Math.Pow(double.Parse(input), 2);
        }

        // Input numbers
        Console.Write("Enter side : ");
        string input = Console.ReadLine();

        // output sum of numbers
        Console.WriteLine("\nArea of square : " + SquareArea(input));
    }
}

File: programme/area-of-square/area-of-square.php
<?
// Write a programme to find area of square

function area_of_square(int|float $num) {

    if (!is_numeric($num)) {

        return 'Invalid input: Input entered was not a number';

    }

    return $num ** 2;

}

$num = readline("Enter a number: ");
$area = area_of_square($num);

print "\nInputted number: {$num}";
print "\nArea of Square: {$area}";
?>